scripts/scil_run_commit.py

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Estimate the fit between a provided tractogram and DWI. Assign a weight to each
```

Comment:
COMMIT means Convex Optimization Modeling for Microstructure Informed Tractography. COMMIT estimates, globally, how a given tractogram explains the DWI in terms of signal fit, assuming a certain forward microstructure model. It assigns a weight to each streamline, which represents how well it explains the DWI signal globally. The default forward microstructure model is stick-zeppelin-ball, which required multi-shell data. It is possible to use the ball-and-stick model for single-shell data.